clang/test/Driver/debug-options.c

```diff
@@ -280,6 +280,15 @@
 // RUN: %clang -### -target %itanium_abi_triple -gmodules -gline-directives-only %s 2>&1 \
 // RUN:        | FileCheck -check-prefix=GLIO_ONLY %s
 //
+// RUN: %clang -### -c -gcas-friendly-debug-info %s -target x86_64-apple-darwin14 2>&1 \
+// RUN:             | FileCheck -check-prefix=G_CASFRIENDLY \
+// RUN:                         -check-prefix=G_DWARF2 \
+// RUN:                         -check-prefix=G_LLDB %s
+// RUN: %clang -### -c -gcas-friendly-debug-info %s -target x86_64-apple-darwin16 2>&1 \
```

Comment:
Nice. Here the triple is good (and necessary) and it won't cause a problem because we're just running the driver which only prints text, and doesn't actually require a backend to be present.